src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -97,6 +97,20 @@ def load_arguments(self, _):
     ui_form_definition_file_type = CLIArgumentType(options_list=['--ui-form-definition'], completer=FilesCompleter(), type=file_type,
                                                    help="A path to a uiFormDefinition file in the file system")
 
+    stacks_name_type = CLIArgumentType(options_list=['--name', '-n'], help='The name of the deployment stack.')
+    stacks_description_type = CLIArgumentType(options_list=['--description'], help='The description of deployment stack.')
+    stacks_stack_type = CLIArgumentType(help='The deployment stack resource id.')
+    stacks_stack_name_type = CLIArgumentType(help='The deployment stack name')
+    stacks_stack_deployment_resource_group = CLIArgumentType(options_list=['--deployment-resource-group', '--dr'], help='The scope at which the initial deployment should be created. If a scope is not specified, it will default to the scope of the deployment stack.')
+    stacks_stack_deployment_subscription = CLIArgumentType(options_list=['--deployment-subscription', '--ds'], help='The scope at which the initial deployment should be created. If a scope is not specified, it will default to the scope of the deployment stack.')
+    stacks_delete_resources_type = CLIArgumentType(arg_type=get_three_state_flag(), options_list=['--delete-resources'], help='Flag to indicate delete rather than detach for the resources.')
+    stacks_delete_resource_groups_type = CLIArgumentType(arg_type=get_three_state_flag(), options_list=['--delete-resource-groups'], help='Flag to indicate delete rather than detach for the resource groups.')
+    stacks_delete_all_type = CLIArgumentType(arg_type=get_three_state_flag(), options_list=['--delete-all'], help='Flag to indicate delete rather than detach for the resources and resource groups.')
+    stacks_deny_settings_mode = CLIArgumentType(arg_type=get_enum_type(['none', 'denyDelete', 'denyWriteAndDelete']), help='Define which operations are denied on resources managed by the stack: denyDelete or denyWriteAndDelete.')
```

Comment:
May I ask could we obtain the enumeration values from the Python SDK?

Reply:
I was added code to obtain the enum values from the Python SDK for **stacks_deny_settings_mode**. Does anything need to be done for stacks_delete_all_type, stacks_delete_resource_groups_type  and stacks_delete_resources_type?

Reply:
I believe adding obtaining the enumeration values from the Python SDK might have resulted in all of the CI build errors. 

This is probably because the stacks sdk is not public yet, will be in a couple of days. 

Before this commit, only the CLI Tests were failing. 

Should I revert this change for now?

Reply:
The error is reported as "ModuleNotFoundError: No module named 'azure.mgmt.resource.deploymentstacks'", it does seem likely that the cause is that the stack sdk in not public.
But it is still suggested to obtain the enumeration values from the sdk, even if the change is reverted for now, it would change for later.
You could omit the ModuleNotFoundError firstly and turn to fix other errors and comments.

Reply:
Sounds good. I'll go ahead and revert the change but I'll leave this conversation unresolved so I can add the change back in once the SDK is public.